core/src/main/resources/lib/layout/layout.jelly

```diff
@@ -159,6 +154,12 @@ ${h.initPageVariables(context)}
       </j:forEach>
     </j:if>
 
+	<j:invokeStatic var="j" className="jenkins.model.Jenkins" method="getActiveInstance" />
+    <j:if test="${j.startupType.name() == 'NEW'}">
+      <script src="${resURL}/${j.installWizardPath}.js" type="text/javascript"/>
+      <link rel="stylesheet" href="${resURL}/${j.installWizardPath}.css" type="text/css" />
```

Comment:
Reason I say it is that means we just have 1 integration point in Jenkins then Vs two (.js and .css).